### PR TITLE
[148] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Constancia2**
contact form
product

### CODE REVIEW

File: submissions/OnyediriConstancia/html-forms/ContactForm.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+<option value="Product">
+<option value="ContactForm">
+<option value="Morehair">
+ </datalist><br><br>
+ <label for="">Write a feedback/Suggestion about our product.</label><br>
+ <textarea name="Comments" id="comments" cols="30" rows="10"></textarea><br><br>
+ 
+<button type="submit">Submit</button>
+</fieldset>
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/html-forms/ContactForm.html:42]: Format properly

**Review by Ireti203 — COMMENTED** [review]
Remove the settings.json file in your html-forms folder

**Comment by Constancia2** [comment]
Pls ma I don't understand the settings . Json , you are talking about. I
can't find it inside

**Comment by Ireti203** [comment]
In the html-forms folder, you have a .vscode file

**Comment by Constancia2** [comment]
Okay

**Comment by Constancia2** [comment]
The problem here is that I have removed the file but I can't push because
the other PR name is interfering. I don't know how to change it to the
former . I used git checkout is telling me the name already exist

File: submissions/OnyediriConstancia/html-forms/ContactForm.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <title>CONTACT FORM</title>
+</head>
+<body>
+   
+    <h1>JUSTHAIR</h1> 
+    <ul>
+        <a href="product.html"> <li> Product</li></a>
+        <a href="ContactForm.html">  <li>ContactForm</li></a>
+        <a href="morehair.html"> <li>MoreHair</li></a> 
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/html-forms/ContactForm.html:15]: Remove whitespace

File: submissions/OnyediriConstancia/html-forms/ContactForm.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<body>
+   
+    <h1>JUSTHAIR</h1> 
+    <ul>
+        <a href="product.html"> <li> Product</li></a>
+        <a href="ContactForm.html">  <li>ContactForm</li></a>
+        <a href="morehair.html"> <li>MoreHair</li></a> 
+        
+    </ul>
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/html-forms/ContactForm.html:17]: Remove whitespace

File: submissions/OnyediriConstancia/html-forms/ContactForm.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <h1>JUSTHAIR</h1> 
+    <ul>
+        <a href="product.html"> <li> Product</li></a>
+        <a href="ContactForm.html">  <li>ContactForm</li></a>
+        <a href="morehair.html"> <li>MoreHair</li></a> 
+        
+    </ul>
+   
+    <p> <strong>  Welcome to JustHair your number one online store for quality lush hair.</strong> </p> 
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/html-forms/ContactForm.html:19]: Remove whitespace

File: submissions/OnyediriConstancia/html-forms/ContactForm.html (hunk 1 above)
Comment (Ireti203) [inline submissions/OnyediriConstancia/html-forms/ContactForm.html:42]: Format properly

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by Constancia2** [comment]
I don't understand the meaning of the white space .

**Pull request merged**